source/table-filters/apply-table-filters.txt

```diff
@@ -45,6 +45,9 @@ Steps
    radio button and enter a value. This applies a limit clause at the 
    end of your SQL query.
 #. Click :guilabel:`Save and close`.
+   After you save your filter, the :guilabel:`Schema model` pane 
+   displays a :icon-fa5:`filter` icon under the :guilabel:`Relational` 
+   header.
```

Comment:
Can we add "for any tables containing a filter" or similar? It's currently a bit vague.

Reply:
Sure thing, this has been updated:

```
   After you save your filter, the Schema model pane 
   displays a filter icon for any tables containing a filter 
   under the Relational header.
 ```